cmd/podman/system/reset.go

```diff
@@ -103,5 +106,11 @@ func reset(cmd *cobra.Command, args []string) {
 		//nolint:gocritic
 		os.Exit(define.ExecErrorCodeGeneric)
 	}
+
+	provider := cmdMach.GetSystemDefaultProvider()
+	err = provider.ResetAllMachines()
+	if err != nil {
+		logrus.Error(err)
+	}
```

Comment:
You have to put this into another file and make it use the same build tags as podman machine and provide a nop function for the unsupported builds.